package.json

```diff
@@ -16,48 +16,48 @@
   },
   "homepage": "https://github.com/bitfinexcom/bfx-report-ui#readme",
   "dependencies": {
-    "@blueprintjs/core": "3.12.0",
-    "@blueprintjs/datetime": "3.7.0",
-    "@blueprintjs/icons": "3.5.1",
-    "@blueprintjs/select": "3.6.0",
+    "@blueprintjs/core": "^3.14.1",
+    "@blueprintjs/datetime": "^3.7.2",
+    "@blueprintjs/icons": "^3.6.0",
+    "@blueprintjs/select": "^3.7.0",
     "@blueprintjs/table": "3.4.1",
     "@blueprintjs/timezone": "3.3.0",
     "classnames": "2.2.6",
-    "dotenv": "^6.2.0",
+    "dotenv": "^7.0.0",
     "flexboxgrid2": "7.2.1",
-    "history": "4.7.2",
-    "i18next": "13.1.4",
-    "i18next-browser-languagedetector": "2.2.4",
-    "i18next-xhr-backend": "1.5.1",
+    "history": "^4.9.0",
+    "i18next": "15.0.7",
+    "i18next-browser-languagedetector": "3.0.1",
+    "i18next-xhr-backend": "2.0.1",
     "lodash": "4.17.11",
     "memoize-one": "5.0.0",
     "moment": "2.24.0",
     "moment-timezone": "0.5.23",
     "node-sass-chokidar": "1.3.4",
     "normalize.css": "8.0.1",
-    "prop-types": "15.6.1",
-    "query-string": "6.2.0",
-    "react": "16.7.0",
-    "react-dom": "16.7.0",
-    "react-i18next": "9.0.4",
-    "react-redux": "6.0.0",
-    "react-router-dom": "4.3.1",
-    "react-scripts": "2.1.3",
-    "react-transition-group": "2.5.3",
+    "prop-types": "^15.7.2",
+    "query-string": "^6.4.0",
+    "react": "^16.8.4",
+    "react-dom": "^16.8.4",
+    "react-i18next": "10.5.2",
+    "react-redux": "^6.0.1",
+    "react-router-dom": "^4.4.0",
```

Comment:
@gasolin i am getting issues with this one, it says: 
`
`` 
npm ERR! code ETARGET
npm ERR! notarget No matching version found for react-router-dom@^4.4.0
```